vulnfeeds/cmd/debian/main.go

```
@@ -0,0 +1,164 @@
+	var pkgInfos []vulns.PackageInfo
+	if value, ok := osvPkgInfos[cveId]; ok {
+		pkgInfos = value
+	}
+	for releaseName, release := range releases {
+		// Skips 'not yet assigned' entries because their status may change in the future.
+		// For reference on urgency levels, see: https://security-team.debian.org/security_tracker.html#severity-levels
+		if release.Urgency == "not yet assigned" {
+			continue
+		}
+		debianVersion, ok := debianReleaseMap[releaseName]
+		if !ok {
+			continue
+		}
+
+		pkgInfo := vulns.PackageInfo{
+			PkgName:   pkgName,
+			Ecosystem: "Debian:" + debianVersion,
+		}
+		pkgInfo.EcosystemSpecific = make(map[string]string)
+
+		if release.Status == "resolved" {
+			if release.FixedVersion == "0" { // not affected
+				continue
+			}
+			pkgInfo.VersionInfo = cves.VersionInfo{
+				AffectedVersions: []cves.AffectedVersion{{Fixed: release.FixedVersion}},
+			}
+		}
+		pkgInfo.EcosystemSpecific["urgency"] = release.Urgency
+		pkgInfos = append(pkgInfos, pkgInfo)
+	}
+	if pkgInfos != nil {
+		osvPkgInfos[cveId] = pkgInfos
+	}
+}
+
+// generateDebianSecurityTrackerOSV converts Debian Security Tracker entries to OSV PackageInfo format.
+func generateDebianSecurityTrackerOSV(debianData DebianSecurityTrackerData) map[string][]vulns.PackageInfo {
+	debianReleaseMap := getDebianReleaseMap()
+	osvPkgInfos := make(map[string][]vulns.PackageInfo)
+	for pkgName, pkg := range debianData {
+		for cveId, cve := range pkg {
+			updateOSVPkgInfos(pkgName, cveId, cve.Releases, osvPkgInfos, debianReleaseMap)
+		}
+	}
+
+	return osvPkgInfos
+}
+
+func writeToOutput(cvePkgInfos map[string][]vulns.PackageInfo) {
+	for cveId := range cvePkgInfos {
+		pkgInfos := cvePkgInfos[cveId]
+		file, err := os.OpenFile(path.Join(debianOutputPathDefault, cveId+".debian.json"), os.O_CREATE|os.O_RDWR, 0644)
+		if err != nil {
+			Logger.Fatalf("Failed to create/write osv output file: %s", err)
```

nit: better to return an error instead, and let the caller handle the error.